src/components/builds/BuildDetails.tsx

```diff
@@ -168,6 +169,9 @@ class BuildDetails extends React.Component<Props> {
           <CardContent>
             <div className={classes.wrapper}>
               <RepositoryNameChip className={classes.chip} repository={build.repository} />
+              <BuildBranchNameChip className={classes.chip} build={build} />
+            </div>
+            <div className={classes.wrapper}>
```

Comment:
Is there a need for an additional wrapper?

Reply:
Design-wise, the task page has 2 rows of chips, this sorts them like that.

Reply:
What do you think of keeping all the chips in a single row? Task page has 6 chips and they just can't fit in one row.

Reply:
Right, but they are in 2 rows there, and users prefer consistant design from what I've seen.